assessment.sql

```diff
@@ -5,24 +5,53 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
-
+Create table licensees(
+  id serial primary key,
+  surname character varying,
+  given_name character varying,
+  gender character varying(1),
+  height integer,
+  weight, integer,
+  dob date,
+  eye_color character varying,
+  hair_color character varying
+);
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT into licensees(surname,given_name,gender,height,weight,dob, eye_color, hair_color)
+VALUES('Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black')
+;
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
+-- **THIS IS PSQL BUT THIS IS HOW WE WERE TAUGHT**
 
--- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
+\copy licensees(surname,given_name,gender,height,weight,dob, eye_color, hair_color) FROM 'data/licensees.csv' WITH( HEADER true, FORMAT csv)
+;
 
--- your code here
 
--- Question 5. Make each of Marilynn Escobar and Chris Whaley one inch shorter. Look them up by their names.
+-- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
--- your code here
+select *
+from licensees
+where eye_color = 'Hazel'
+  and hair_color = 'Brown'
+  or hair_color = 'Black'
+  ;
```

Comment:
This query has an operator precedence problem. It will retrieve all users with a 'Black' hair color regardless of eye color. Use parentheses to force a different order of evaluation. 4/5